tests/foreman/cli/test_ansible.py

```diff
@@ -525,8 +529,9 @@ def test_positive_install_ansible_collection(self, rex_contenthost, target_sat):
 
         :expectedresults: Ansible collection can be installed on content host via REX.
         """
-        client = rex_contenthost
+        client = rhel_contenthost
         # Enable Ansible repository and Install ansible or ansible-core package
+        client.register(module_org, None, module_ak_with_cv.name, target_sat)
         client.create_custom_repos(rhel8_aps=settings.repos.rhel8_os.appstream)
```

Comment:
Please update here to fetch the repos as per the rhel version.

Reply:
@shweta83 Updated